generate_cendl.py

```diff
@@ -89,47 +88,44 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
 
     # Extract files
 
-    suffix = ''
     with zipfile.ZipFile(f) as zf:
         print('Extracting {0}...'.format(f))
-        zf.extractall(path=os.path.join(endf_files_dir, suffix))
+        zf.extractall(path=endf_files_dir)
 
 
 # ==============================================================================
 # GENERATE HDF5 LIBRARY -- NEUTRON FILES
 
 # Get a list of all ENDF files
-neutron_files = glob.glob(release_details[args.release]['neutron_files'])
+neutron_files = release_details[args.release]['neutron_files']
 
 # Create output directory if it doesn't exist
-if not os.path.isdir(args.destination):
-    os.mkdir(args.destination)
+args.destination.mkdir(parents=True, exist_ok=True)
```

Comment:
Again this is fine - it extends the original behaviour by automatically creating any parent directories.